web/src/features/panels/LeftPanel.tsx

```diff
@@ -17,26 +17,6 @@ import RankingPanel from './ranking-panel/RankingPanel';
 
 import ZoneDetails from './zone/ZoneDetails';
 
-// Remove index.html from URL
-function RedirectIndexWrapper({ children }: { children: JSX.Element }) {
-  const [searchParameters] = useSearchParams();
-  const location = useLocation();
-
-  if (location.pathname.includes('/index.html')) {
-    const pathWithoutIndex = location.pathname.replace('/index.html', '');
-    return (
-      <Navigate
-        to={{
-          pathname: pathWithoutIndex,
-          search: searchParameters.toString(),
-        }}
-      />
-    );
-  }
-
-  return children;
-}
-
```

Comment:
Honestly I'm glad to see this is gone 😅

Reply:
Haha yeah same, is was quite annoying that we had to add this 😬